docs/overview.md

```diff
@@ -170,7 +165,7 @@ Artefacts in Tremor are declarative specifications of:
 - Pipelines - A pipeline specification is a specific configuration of a pipeline graph
 - Bindings - A binding specification describes how onramps, pipelines and offramps should be interconnected
 
-Artefacts can be thought of analagously to code. They are a set of instructions, rules or configurations. As such they are registered with Tremor via its API and stored in Tremor's artefact repository.
+Artefacts can be thought of analagous to code. They are a set of instructions, rules or configurations. As such they are registered with Tremor via its API and stored in Tremor's artefact repository.
 
 Deployment in Tremor, is achieved through a mapping artefact. The mapping artefact specifies how artefacts should be deployed into one or many runtime instances, activated, and connected to live instances of onramps or offramps.
```

Comment:
"Deployment in Tremor is achieved through a mapping artefact." No need for the comma, right? :D